user-console/src/main/java/org/pentaho/mantle/client/solutionbrowser/SolutionBrowserPanel.java

```diff
@@ -823,4 +826,13 @@ private static final native String encodeUri( String URI )
     return encodeURIComponent(URI);
   }-*/;
 
+  private class InfoDialog extends PromptDialogBox { // All of this is to get a OK button that is aligned right
```

Comment:
Why not put the InfoDialogBox in the commons-gwt project next to MesssageDialogBox, so others can use it besides the SolutionsBrowserPanel?

Reply:
This is a one off, does not pertain to other dialogs